atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/StreamStoreRenderer.java

```diff
@@ -140,7 +139,6 @@ protected void run() {
                 line();
                 importRenderer.renderImports();
                 line();
-                line("@Generated(\"",  StreamStoreRenderer.class.getName(), "\")");
```

Comment:
This was recently added by Carrino, probably want to keep these generated annotations and sync upstream.